concepts/inheritance/introduction.md

```diff
@@ -25,13 +25,19 @@ Consider an animal named `Lion`, having a class like,
 //Lion class is a child class of Animal.
 public class Lion extends Animal {
 
+    @Override
     public void bark() {
         System.out.println("Lion here!!");
     }
 
 }
 ```
 
+```exercism/note
```

Comment:
I believe the admonitions should be prefixed by `~~~~` instead of three backtics

Reply:
Just in case I copied it from https://github.com/exercism/java/blob/832c70349b3ce74f36f046cd88c6a0ffacff3452/concepts/datetime/introduction.md?plain=1#L37

Maybe is wrong there as well?

Reply:
Done!

Reply:
Hmm thanks for pointing that out. I double checked the [page on the website](https://exercism.org/tracks/java/concepts/datetime) and it seems to render correctly with three backticks as well. Still, the [documentation](https://exercism.org/docs/building/markdown/markdown#h-special-blocks-sometimes-called-admonitions) says to use `~~~~` so let's follow that.